src/sections/CareersHero/index.css

```diff
@@ -0,0 +1,78 @@
+@import "../../styles/all.css";
+
+.root {
+  padding-bottom: $spacing-224;
+
+  @include Breakpoint-tabletOnly {
+    padding-bottom: $spacing-136;
+  }
+
+  @include Breakpoint-mobileOnly {
+    padding-bottom: $spacing-136;
+  }
+}
+
+.title-desktop {
+  display: flex;
+  flex-direction: column;
+
+  margin-top: calc($spacing-224 - $navbar-height);
+  margin-bottom: $spacing-136;
+
+  @include Breakpoint-tabletAndBelow {
+    display: none;
+  }
+}
+
+.title-mobile {
```

Comment:
Add a max-width here to get 3 lines (vw unit)

Reply:
max-width: $mobile-padding; 
4vw is good?

Reply:
`$mobile-padding`? 
`100vw = 100% of the viewport`, so `80vw` I think it's enough

Reply:
It was a constant defined on layout.css.
Ok.

Reply:
Yes, but it's not for `max-width` I think, probably it's only for some specific sections because I don't remember that `$mobile-padding` :/